backend/test/hooks.js

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
```

Comment:
Is this used?

Reply:
Think its needed - I get a Mongoose error: _The uri` parameter to `openUri()` must be a string, got "undefined"_ if I remove it

Reply:
Fair enough, maybe look into why it's doing that separately then